pkg/storage/export/export/export.go

```diff
@@ -1394,7 +1396,7 @@ func (ctrl *VMExportController) expandVirtualMachine(vm *virtv1.VirtualMachine)
 		return vm, nil
 	}
 
-	conflicts := ctrl.instancetypeMethods.ApplyToVmi(field.NewPath("spec", "template", "spec"), instancetypeSpec, preferenceSpec, &vm.Spec.Template.Spec, &vm.Spec.Template.ObjectMeta)
+	conflicts := instancetypeapply.NewVMIApplier().ApplyToVMI(field.NewPath("spec", "template", "spec"), instancetypeSpec, preferenceSpec, &vm.Spec.Template.Spec, &vm.Spec.Template.ObjectMeta)
```

Comment:
This should be using an interface to decouple it from the concrete implementation.

Reply:
this will be removed completelly when https://github.com/kubevirt/kubevirt/pull/13796 is merged

Reply:
Can you drop these changes from the PR for now?